test/plots/index.ts

```diff
@@ -46,6 +45,7 @@ export * from "./cars-hexbin.js";
 export * from "./cars-jitter.js";
 export * from "./cars-mpg.js";
 export * from "./cars-parcoords.js";
+export * from "./channel-domain.js";
```

Comment:
I’m following the principle here that test names should indicate which features are being tested, rather than tests being feature-agnostic examples. We can have the latter, too, but here the tests are intended to test imputed channel domains so it’s better to call them that.